### PR TITLE
chore: update content on fast-animation readme

### PR DESCRIPTION
**Pull request by Falkicon**
## Description
Updated markdown formatting and minor edits to grammar, sentence structure, and punctuation.

## Issue type checklist
- [x] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
- [ ] **New feature**: non-breaking change that adds functionality.
- [ ] **Breaking change**: fix or feature that will cause current functionality to change.

## Process & policy checklist

- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the CONTRIBUTING documentation and followed the standards set for this project.

 
Click here for details on [formatting pull requests](https://microsoft.github.io/fast-dna/docs/contributing/working#formatting-messages).

### CODE REVIEW

File: packages/fast-animation/README.md [hunk 1]
```diff
@@ -30,7 +33,7 @@ var mySequence = new AnimateSequence([
     new AnimateTo(element3, { x: 20 })
 ]);
 
-// Attach an onFinish callback
+// Attach an onFinish callback.
```
Comment (nalogiudice) [inline packages/fast-animation/README.md]: nit: this period feels funny to me. Is this the new way we want to do comments? Most comments feel like statements most of the time to me

Comment (chrisdholt) [reply packages/fast-animation/README.md]: This does feel more like a fragment

Comment (Falkicon) [reply packages/fast-animation/README.md]: But it is not a fragment, it has an independent clause with a subject, a verb, and a complete thought.
Subject "onFinish callback" and a verb "Attach".

File: packages/fast-animation/README.md [hunk 2]
```diff
@@ -88,21 +99,26 @@ var myAnimationSequence = new AnimateSequence([
...
+#### Life cycle hooks for AnimateGroup and AnimateSequence
 
-#### Life cycle hooks
 `onFinish: { () => void }` If provided, `onFinish` will be called when the animation has completed.
 
 ## Running the localhost testing environment
-- Run `npm run dev-server:react` to view the react examples (localhost:9005).
-- Run `npm run build` to compile files to the build folder.
+
+* Run `npm run dev*server:react` to view the react examples (localhost:9005).
```
Comment (nicholasrice) [inline packages/fast-animation/README.md]: Should be `npm run dev-server:react`

Comment (Falkicon) [reply packages/fast-animation/README.md]: fixed.

**Pull request merged**